src/transformers/generation_tf_utils.py

```diff
@@ -893,12 +917,21 @@ def _generate_beam_search(
     def _reorder_cache(past, beam_idx):
         return tuple(tf.gather(layer_past, beam_idx, axis=1) for layer_past in past)
 
-    def adjust_logits_during_generation(self, logits, **kwargs):
+    def adjust_logits_during_generation(
+        self, logits, cur_len, max_length, forced_bos_token_id, forced_eos_token_id, **kwargs
+    ):
```

Comment:
Since there are no `LogitsProcessor`s in TF `generate`, keeping the forced bos and eos token logic here.

Reply:
Cool! Very much agree here - until `generate()` is not refactored, this looks like a very good solution